examples/powertools-examples-core/cdk/README.md

```diff
@@ -0,0 +1,64 @@
+#  Powertools for AWS Lambda (Java) - Core Utilities Example
```

Comment:
We're missing a powertools-examples-core/README.md to link down to the concrete examples, and provide probably some high level guidance on which one to consider

Reply:
Also the first part of this doesn't mention the CDK at all. Probably something for the title even?